src/app/templates/app/dots.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'app/base.html' %} {% load static %} {% block dots %}
+<iframe
+  src="http://151.145.81.233"
```

Comment:
@ilans What feature does this dots iframe provides for the web tools?

Reply:
Hi Rohit, "dots" is an open source visual SPDX tool for constructing and editing SPDXv3 documents. I created it as a playground for newcomers, but it could be useful for advanced users too. Give it a try...

I presented it at the technical and implementers meetings and was asked to add it to the tools website, which is what it was made for in the first place. The reason it's embedded in an iframe is that it's a React app (a pretty big one). It's also in active development and lives in its own repo: https://github.com/condots/dots

Reply:
Note that I've changed the iframe src to embed dots using https.

Reply:
Can I merge?

Reply:
@ilans @goneall Thanks for confirming. Left one more comment on the new commit to address before merge. Other changes lgtm.